### ISSUE TITLE
Is this project dead ? If so, why ?

### ISSUE THREAD

**Issue by sfermigier** [deliveryhero/lymph#308]

**Comment by dushr** [comment]
@sfermigier Yup, unfortunately this project is dead. If you're interested in a similar project, I would suggest you take a look at [nameko](https://www.nameko.io/)
